### PR TITLE
[10.0]-[MIG]-Migrated module from v8 to v10 Business_requirement_deliverable_project_task_categ

### PR DESCRIPTION
**Pull request by YogeshMahera-SerpentCS**
#48 
- Business_requirement_deliverable_project_task_categ

### CODE REVIEW

**Comment by YogeshMahera-SerpentCS** [comment]
**Notes:-** 
- effective_hours field
 - In v8 that field is available in project module but later on v9 it's moved on `project_timesheet`,
 now in v10, it's again moved to hr_timesheet.

- So added hr_timesheet dependency in `business_requirement_deliverable_project` project.

**Comment by elicoidal** [comment]
@YogeshMahera-SerpentCS I tested it but could not find the categories anywhere in the Resources lines. Did I miss something?

**Comment by elicoidal** [comment]
@YogeshMahera-SerpentCS

**Comment by YogeshMahera-SerpentCS** [comment]
@elicoidal 
I think this PR depends on #231(BR Project module)
So We have to wait for merge.

**Comment by elicoidal** [comment]
OK: let's try to fix all other PR first :smile:

**Comment by elicoidal** [comment]
I think you can rebase this one as the data model in #231 should be stable now

**Comment by YogeshMahera-SerpentCS** [comment]
rebased

**Comment by elicoidal** [comment]
should still wait for project to be merged

**Comment by elicoidal** [comment]
@YogeshMahera-SerpentCS can you rebase as #231 is merged so that we can finish this one?

**Comment by YogeshMahera-SerpentCS** [comment]
@elicoidal rebased

**Comment by elicoidal** [comment]
Thanks @YogeshMahera-SerpentCS
Seems travis and runbot are both failing: can you have a look?

**Comment by martintorresvm** [comment]
@YogeshMahera-SerpentCS 

ParseError: "categ_id" while parsing /home/travis/build/OCA/business-requirement/business_requirement_deliverable_project_task_categ/data/business_requirement_deliverable_project_task_categ_demo.xml:20, near

**Comment by YogeshMahera-SerpentCS** [comment]
Yes I already checked this one in  my local it's seems everything OK. but I am going to check deep.

**Comment by YogeshMahera-SerpentCS** [comment]
@elicoidal @victormartinelicocorp Could you please review it

File: business_requirement_deliverable_project_task_categ/models/business_requirement_resource.py [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-from openerp import api, fields, models
+from odoo import api, fields, models
 
 
 class BusinessRequirementDeliverableCateg(models.Model):
     _inherit = "business.requirement.resource"
 
     categ_id = fields.Many2one(
-        'project.category.main',
+        'project.tags',
```
Comment (martintorresvm) [inline business_requirement_deliverable_project_task_categ/models/business_requirement_resource.py]: @YogeshMahera-SerpentCS Please use this reference and update related:

https://github.com/OCA/project/blob/10.0/project_task_category/models/project_task.py#L14

**Review by martintorresvm — CHANGES_REQUESTED** [review]

**Review by elicoidal — COMMENTED** [review]
Are they tests?

**Comment by YogeshMahera-SerpentCS** [comment]
@victormartinelicocorp Could you please check it

**Review by elicoidal — APPROVED** [review]
Functional tests OK

**Comment by elicoidal** [comment]
@victormartinelicocorp

**Review by martintorresvm — APPROVED** [review]
LGTM

**Pull request merged**